pkg/agent/multicast/mcast_discovery.go

```diff
@@ -53,11 +55,12 @@ var (
 )
 
 type IGMPSnooper struct {
-	ofClient   openflow.Client
-	ifaceStore interfacestore.InterfaceStore
-	eventCh    chan *mcastGroupEvent
-	Validator  types.MulticastValidator
-	anpEnabled bool
+	ofClient          openflow.Client
+	ifaceStore        interfacestore.InterfaceStore
+	eventCh           chan *mcastGroupEvent
+	validator         types.MulticastValidator
+	anpEnabled        bool
+	IGMPReportNPStats map[v1beta2.NetworkPolicyType]map[apitypes.UID]map[string]*types.RuleMetric
```

Comment:
Please add some comments about the map, esp. what is the key for each nested map.

Maybe more efficient to define three separate maps for each NP type respectively.

Reply:
Does the field need to be public? If not igmpReportNPStats

Reply:
The variable is in lower case here but in upper case when used in methods, can this commit even be compiled successfully?

Reply:
I don't see any existence of field IGMPReportNPStats from the commit https://github.com/antrea-io/antrea/commit/7bd8781325034f78e12ff56538c449e095439de7 you were referring to.

Reply:
It was in fe0697f when I was reviewing

Reply:
my bad. will keep a close check before pushing